data-raw/internal_data.R

```diff
@@ -58,6 +58,16 @@ required_meta_cols <- c(
   "item_type"
   )
 
+idx_cols <- c(
```

Comment:
A small description why it is needed and where it is used would be helpful here. What is 'idx'?

Reply:
Done.